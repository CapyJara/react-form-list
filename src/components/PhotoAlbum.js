import React from 'react';
import PropTypes from 'prop-types';
import Photos from './Photos';

function PhotoAlbum(photos) {
  return (
    <ul>
      <Photos photos={photos} />
    </ul>
  );
}

PhotoAlbum.propTypes = {
  photos: PropTypes.array.isRequired
};

export default PhotoAlbum;
