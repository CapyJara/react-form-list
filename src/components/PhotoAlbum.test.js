import React from 'react';
import { shallow } from 'enzyme';
import PhotoAlbum from './PhotoAlbum';

describe('PhotoAlbum component', () => {
  it('renders a PhotoAlbum', () => {
    const photos = [
      {}
    ]

    const wrapper = shallow(<PhotoAlbum photos={photos} />);
    expect(wrapper).toMatchSnapshot();
  });
});
