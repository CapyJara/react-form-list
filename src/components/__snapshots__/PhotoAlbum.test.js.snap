// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoAlbum component renders a PhotoAlbum 1`] = `
<ul>
  <Photos
    photos={
      Object {
        "name": "Red",
        "rgb": Object {
          "blue": 0,
          "green": 0,
          "red": 255,
        },
      }
    }
  />
</ul>
`;
